superset/migrations/versions/b8d3a24d9131_new_dataset_models.py

```diff
@@ -0,0 +1,173 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""new dataset models
+
+Revision ID: b8d3a24d9131
+Revises: aea15018d53b
+Create Date: 2021-11-11 16:41:53.266965
+
+"""
+
+from uuid import uuid4
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy_utils import UUIDType
+
+from superset import db
+from superset.connectors.sqla.models import SqlaTable
```

Comment:
Don't import models directly in migrations. If (when) we ever update this model, the migrations might (will) break.

Reply:
Ah, great point. Let me fix it.